projects/observability/src/shared/dashboard/widgets/top-n/top-n-widget.model.ts

```diff
@@ -7,10 +7,10 @@ import {
   ModelProperty,
   ModelPropertyType
 } from '@hypertrace/hyperdash';
-import { ModelInject, MODEL_API } from '@hypertrace/hyperdash-angular';
+import { MODEL_API, ModelInject } from '@hypertrace/hyperdash-angular';
```

Comment:
something is wrong with our lint checks? How come this was not caught before? 🤔

Reply:
might be my IDE. updated

Reply:
This was correct as is - make sure your IDE is using the same tslint as the project (this changed in newer versions of tslint, but all files should be up to date0

Reply:
I think this is a side effect of working in the parent project in the IDE.

In a related topic, do we really have to care about how the imports are ordered? IMO its just another headache that we can avoid.

Reply:
That could be, depending on the IDE. For my VS code, linting + testing only works for the main project, not submodules (that's one of the main reasons I work directly against the individual repos).

The import ordering should be more or less a no-op, since it's autofixed by both prettier and tslint, both of which should be running. It's part of pretty much every style guide I've ever read to keep things consistent make it easier to read, and lower the diff sizes of changes. If it were a burdensome rule, I could see the arg for removing it - but here, I think the issue is just that the linter isn't running, which means more critical rules would be missed too.

Reply:
Its not really a burdensome rule. But in this case, the import optimization doesn't work properly with tslint in the IDE. Spending time to then fix it to order it when it gets messed up is burdensome IMO. Hence the question about if its really necessary.

Reply:
I think we should focus on getting tslint working in the IDE (or documenting the expected dev flow), since I think linting in general becomes quite burdensome and not practical without IDE integration.